content/id/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Kustomisasi Service DNS
+content_type: task
+min-kubernetes-server-version: v1.12
+---
+
+<!-- overview -->
+Laman ini menjelaskan cara mengonfigurasi DNS
+{{< glossary_tooltip text="Pod" term_id="pod" >}} kamu dan menyesuaikan
+proses resolusi DNS pada klaster kamu.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Klaster kamu harus menjalankan tambahan (_add-on_) CoreDNS terlebih dahulu.
+[Migrasi ke CoreDNS](/id/docs/tasks/administer-cluster/coredns/#migrasi-ke-coredns)
+menjelaskan tentang bagaimana menggunakan `kubeadm` untuk melakukan migrasi dari `kube-dns`.
+
+{{% version-check %}}
+
+<!-- steps -->
+
+## Pengenalan
+
+DNS adalah Service bawaan dalam Kubernetes yang diluncurkan secara automatis
+melalui _addon manager_
+[add-on klaster](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/README.md).
+
+Sejak Kubernetes v1.12, CoreDNS adalah server DNS yang direkomendasikan untuk menggantikan kube-dns. Jika klaster kamu
+sebelumnya menggunakan kube-dns, maka kamu mungkin masih menggunakan `kube-dns` daripada CoreDNS.
+
+{{< note >}}
+Baik Service CoreDNS dan kube-dns diberi nama `kube-dns` pada _field_ `metadata.name`.
+Hal ini agar ada interoperabilitas yang lebih besar dengan beban kerja yang bergantung pada nama Service `kube-dns` lama untuk me-_resolve_ alamat internal ke dalam klaster. Dengan menggunakan sebuah Service yang bernama `kube-dns` mengabstraksi detail implementasi yang dijalankan oleh penyedia DNS di belakang nama umum tersebut.
```

Comment:
should we standardize "workload" with "beban kerja"? I think we can

Reply:
As long as not API name, I think we can.